language-configuration.json

```diff
@@ -13,18 +13,24 @@
     ],
     // symbols that are auto closed when typing
     "autoClosingPairs": [
-        ["{", "}"],
-        ["[", "]"],
-        ["(", ")"],
-        ["\"", "\""],
-        ["'", "'"]
+         // Should not autoclose in char literals,
+         // but the only option is string literals, so that'll have to do
+        { "open": "{", "close": "}", "notIn": ["string"] },
+        { "open": "{-", "close": "-}", "notIn": ["string"] },
+        { "open": "[", "close": "]", "notIn": ["string"] },
+        { "open": "(", "close": ")", "notIn": ["string"] },
+        { "open": "\"", "close": "\"", "notIn": ["string"] },
```

Comment:
Should this be _only_ in strings maybe?

Reply:
these pairs should not be auto-closed inside strings, mainly because char literals are treated as strings and there's no separate category for char literals.

not sure what you're asking?